### ISSUE TITLE
Run Lighthouse for accessibility and performance testing

### ISSUE THREAD

**Issue by allanhaggett** [allanhaggett/wp-bcgov-corp-learn-portal#20]

**Comment by allanhaggett** [comment]
![image](https://user-images.githubusercontent.com/47408983/143507970-859b63bf-a8a6-435d-a22e-645620768ddf.png)
I'm working on tweaks to bring performance up into the 90s by adding image heightxwidth in some place where it's currently missing, and I can also bring the accessibility score up by applying a label in some place that's missing.
